planner/core/logical_plan_builder.go

```diff
@@ -659,7 +676,10 @@ func (b *PlanBuilder) buildProjection(p LogicalPlan, fields []*ast.SelectField,
 		p = np
 		proj.Exprs = append(proj.Exprs, newExpr)
 
-		col := b.buildProjectionField(proj.id, schema.Len()+1, field, newExpr)
+		col, err := b.buildProjectionField(proj.id, schema.Len()+1, field, newExpr)
+		if err != nil {
+			return nil, 0, errors.Trace(err)
```

Comment:
Ditto

Reply:
I think we'd better left `errors.Trace` because there may be more error returned by `buildProjectionField` not with stack in the future? I'm not sure.